Helper/AlgoliaHelper.php

```diff
@@ -95,7 +81,7 @@ public function __construct(
     public function createClient(): void
```

Comment:
Does this need to be public if only called by `AlgoliaHelper`? As it manages internal state I'm not sure it would be a good candidate for a possible future plugin.

Reply:
good call, updating it